components/ui/form/Schedule.tsx

```diff
@@ -62,6 +65,34 @@ const TimeRangeField = ({ name }: TimeRangeFieldProps) => {
     );
   }, []);
 
+  const [valueSelectStart, setValueSelectStart] = useState<Option>(initialValueforSelectStart);
+
+  const createDateAndModifyNewValue = (onChange: any, option: any) => {
+    onChange(new Date(option?.value as number));
+    setValueSelectStart(option);
+  };
+
+  const calculateTimeRange = (value: Option) => {
```

Comment:
I noticed this function returns a **boolean** value, so what do you think if you renamed this by a name with prefix 'is' or 'has'? Take a look at the naming conventions for boolean functions or methods.